LSP.sublime-settings

```diff
@@ -134,6 +134,9 @@
   // "phantom" - show a phantom on the top when code actions are available
   "show_code_lens": "annotation",
 
+   // Inlay hints are short annotations within the code, which show variable types or parameter names.
+  "show_inlay_hints": true,
```

Comment:
I would prefer this to be set to false by default. The phantoms don’t behave perfectly UX-wise. Furthermore there’s a clear divide in preference for this feature by users; we can see some issues in LSP-rust-analyzer by users asking how to turn this off.